R/iterate-responses.R

```diff
@@ -0,0 +1,34 @@
+#' Tools for working with lists of responses
+#'
+#' * `resps_combine()` combines the data from each responses into a single
+#'   object.
+#' * `resps_response()` returns all successful responses.
+#' * `resps_error()` returns all errors.
+#'
+#' @export
+#' @param resps A list of responses (possibly including errors).
+#' @param resp_data A function that takes a response (`resp`) and
+#'   returns its data as a vector or data frame.
```

Comment:
```suggestion
#'   returns it's data as a vector or data frame.
```

Reply:
Although I often get this wrong I'm pretty sure I'm right here 😄

Reply:
But maybe the wording is confusing.

Reply:
Ah, so true, you're right, it's "its" 😊